src/pages/dar_application/DataAccessRequest.jsx

```diff
@@ -162,7 +162,7 @@ export default function DataAccessRequest(props) {
             isAsync={true}
             isMulti={true}
             title={'2.1 Select Dataset(s)'}
-            titleStyle={titleStyle}
+            titleStyle={readOnlyMode ? {...titleStyle, ...noTopMarginStyle} : titleStyle}
             validators={[FormValidators.REQUIRED]}
```

Comment:
Without updating these styles, there was a really big gap between the accordion title an the content.